en/navigation_v2.ftl

```diff
@@ -79,3 +81,6 @@ navigation-v2-webassembly = { -brand-name-webassembly }
 navigation-v2-learn-more-about-the-new = Learn more about the new, low-level, assembly-like language.
 navigation-v2-mozilla-innovation-projects = { -brand-name-mozilla } Innovation Projects
 navigation-v2-discover-ways-to-bring = Discover ways to bring bright ideas to life.
+navigation-v2-mozilla-ai = { -brand-name-mozilla-ai }
+
+navigation-v2-mozilla-ventures = { -brand-name-mozilla-ventures }
```

Comment:
I wonder whether these two should be removed for now as not to surprise anyone — the menu items were not actually added in the end.

Reply:
@stephaniehobson who can make this decision? Should we keep them and they will show up eventually?

Reply:
For now, it doesn't seem like they're going to be added to the nav anytime soon, so I'm going to remove them from our codebase for now. Thanks for the call-out @janbrasna !